OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -452,7 +452,7 @@ Action InitInputPanel(Widget panel)
 				};
 
 				var header = ScrollItemWidget.Setup(hotkeyHeader, returnTrue, doNothing);
-				header.Get<LabelWidget>("LABEL").GetText = () => "Game Commands";
+				header.Get<LabelWidget>("LABEL").GetText = () => "Game commands";
```

Comment:
Why are you changing the capitalization here?  These are titles, so should be in Title Case.